index.html

```diff
@@ -12,17 +12,18 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <base href="/">
```

Comment:
Good point. It's useful enough for this case, though, that I'm OK with usurping it. If someone has a different need for `<base>`, I think it's reasonable to put the onus on them to come up with a different scheme for differential serving.